### ISSUE TITLE
Design a logo for Jugaadfest

### ISSUE THREAD

**Issue by mariobehling** [fossasia/jugaadfest.com#3]
Please help us to design a logo for Jugaadfest. We would like a simple logo preferable in a vector format (if possible svg) that embodies the idea of Jugaad as described below.

**Background**: Jugaad (alternatively Juggaar) is a colloquial Hindi (Devanagiri: जुगाड़, Latin: Jugāṛ), Urdu (Urdu alphabet: جگاڑ, Latin: Jugāṛ) and Punjabi word, which has various meanings depending on the situation. Roughly translated, jugaad is a "hack". It could also refer to an innovative fix or a simple work-around, a solution that bends the rules, or a resource that can be used in such a way. It is also often used to signify creativity—to make existing things work, or to create new things with meager resources. Jugaad is increasingly accepted as a management technique and is recognized all over the world as an acceptable form of frugal engineering at peak in India. https://en.wikipedia.org/wiki/Jugaad

@victoria-bondarchuk @spMohanty

**Comment by imujjwal96** [comment]
![img-20170414-wa0007](https://cloud.githubusercontent.com/assets/8245662/25049159/f878c4c6-215d-11e7-8914-c42595153dbc.jpg)

I have created this vague design. Some inputs would be appreciated

**Comment by harshit98** [comment]
@mariobehling I have created some rough designs. 
![selection_002](https://cloud.githubusercontent.com/assets/22245418/25053517/5bc0d87c-2175-11e7-84bc-7b39cb967736.png)
![selection_004](https://cloud.githubusercontent.com/assets/22245418/25053524/62144b0a-2175-11e7-952e-cbf7569562b6.png)

**Comment by AliAyub007** [comment]
![jugaad fest 2](https://cloud.githubusercontent.com/assets/25840461/25071077/da73e710-22c7-11e7-9dcb-2e752d17c3da.png)

**Comment by KukkadapuSushma** [comment]
I just done a simple design : 

![jugaad-logo](https://cloud.githubusercontent.com/assets/14278142/25072556/243abf32-22ef-11e7-9569-9a72a93cdc90.jpg)

**Comment by meets2tarun** [comment]
I have created this design.
Any feedback is highly helpful.
![jugaad](https://cloud.githubusercontent.com/assets/15213246/25100469/d24d7450-23cd-11e7-8910-f6f814db3825.jpg)

**Comment by khushmeen** [comment]
Hey @mariobehling @geekyd told me about the logo requirement
Here is the logo. I hope you'll like it. Thanks

![jf2](https://cloud.githubusercontent.com/assets/27868552/25293446/93aa677a-26f8-11e7-88f0-59a509412941.jpg)

**Comment by kubami9** [comment]
First on dark background, second on transparent, third on white
![logo_dark](https://user-images.githubusercontent.com/16598854/33518586-6574f2f8-d797-11e7-86cc-e997bb5c9fb6.png)
![logo_jugaadfest](https://user-images.githubusercontent.com/16598854/33518587-6592f2e4-d797-11e7-9bec-5114bfad2e90.png)
![logo_white](https://user-images.githubusercontent.com/16598854/33518588-65b1cd5e-d797-11e7-8a2a-494ab967bf8a.png)

**Comment by Ploypiti** [comment]
I made 2 versions of the design, one with an Urdu letter and another in plain English, feedbacks are highly appreciated, which one do you like the most?
![jugaad logo preview](https://user-images.githubusercontent.com/32614202/34026982-51d5ba76-e18c-11e7-8c16-706af0d13c92.jpg)
![jugaad logo preview dark bg](https://user-images.githubusercontent.com/32614202/34027003-6d8dea2c-e18c-11e7-8ebb-e81601e4f94b.jpg)
![jugaad logo scheme](https://user-images.githubusercontent.com/32614202/34027010-859583c8-e18c-11e7-81eb-07ae89e93f46.jpg)
Also SVG file  ⬇️ 
[JUGAAD logo.zip](https://github.com/fossasia/jugaadfest.com/files/1561604/JUGAAD.logo.zip)

**Comment by 0x48piraj** [comment]
Chose a motto : "**Hacking via Indian Way**" :heart: 
> **Created many mock-ups. Hope you all like it.** :hear_no_evil: 

![1](https://user-images.githubusercontent.com/5800726/34078249-88e87bbe-e33c-11e7-90b6-12b115c36a08.PNG)
![2](https://user-images.githubusercontent.com/5800726/34078250-893a5510-e33c-11e7-9820-d3d08583bd40.PNG)
![3](https://user-images.githubusercontent.com/5800726/34078251-899141ea-e33c-11e7-8778-f401a1ae9bc0.PNG)
![4](https://user-images.githubusercontent.com/5800726/34078252-89e031e2-e33c-11e7-9106-480d01217865.PNG)
![5](https://user-images.githubusercontent.com/5800726/34078253-8a2eced8-e33c-11e7-86bb-39345145650f.PNG)
![6](https://user-images.githubusercontent.com/5800726/34078254-8a866bb6-e33c-11e7-8af7-f1c33873804e.PNG)
![7](https://user-images.githubusercontent.com/5800726/34078255-8ad7de88-e33c-11e7-9c57-4df9db4738fa.PNG)
![8](https://user-images.githubusercontent.com/5800726/34078256-8b27d672-e33c-11e7-93a2-c04930cb5575.PNG)
![9](https://user-images.githubusercontent.com/5800726/34078258-8b723d48-e33c-11e7-8fd9-4f8d920fe4ed.PNG)
![10](https://user-images.githubusercontent.com/5800726/34078259-8bc13376-e33c-11e7-8ca2-d1425b835f3b.PNG)

---

# Surprise! Surprise!
![extra](https://user-images.githubusercontent.com/5800726/34078269-abd9f878-e33c-11e7-961e-3c06d573e528.PNG)

**Comment by anshumanv** [comment]
@mariobehling do check this out! 
Some amazing piece of work by students. :smile: :sparkles:

**Comment by YashKumarVerma** [comment]
# Logo
![image](https://user-images.githubusercontent.com/14032427/34860422-4a151e3c-f783-11e7-8817-895915757030.png)
Where do i share the editable files ?

# Asssets:
Logo SVG : http://lsppublicschool.org/yashkumarverma-host/vault/logo1.svg
Logo item svg : http://lsppublicschool.org/yashkumarverma-host/vault/raw%20background.svg

# Creation
Made with ❤️ using Inkscape.
http://lsppublicschool.org/yashkumarverma-host/vault/Screenshot%20(299).png

**Comment by 0x48piraj** [comment]
*Seeing these all Cool Designs, My inner designer was once again pumped.*

> Here are some more mockups. 
I will further improve these, if any of my designs are selected.

### The motto was devised by me "Hacking, the Indian way!" :relaxed: 
*(Thanks for using it in your designs)*

Ping @mariobehling :heart_decoration: 

#### Here we go - 

![image](https://user-images.githubusercontent.com/5800726/34863203-25e56114-f794-11e7-9f5a-450efb7c2c62.png)

---

![image](https://user-images.githubusercontent.com/5800726/34863265-7723bf08-f794-11e7-94d4-0db160b563f6.png)

---

![image](https://user-images.githubusercontent.com/5800726/34863282-8e7e1522-f794-11e7-8739-736bde9bdffd.png)

---

![image](https://user-images.githubusercontent.com/5800726/34863271-82992332-f794-11e7-8a2d-cc664e124e57.png)

---

![image](https://user-images.githubusercontent.com/5800726/34863328-e2f443ce-f794-11e7-89df-f12ed731f0b3.png)

---

![image](https://user-images.githubusercontent.com/5800726/34863334-ee56019e-f794-11e7-80b8-9f8336a55816.png)

---

![image](https://user-images.githubusercontent.com/5800726/34863343-fd0aa884-f794-11e7-8d88-661204e98af2.png)

---

![image](https://user-images.githubusercontent.com/5800726/34863365-18248c7a-f795-11e7-99bd-c1a86c2eff90.png)

---

![image](https://user-images.githubusercontent.com/5800726/34863369-1ca9122a-f795-11e7-813b-fd3fe8c18fcd.png)

---

![image](https://user-images.githubusercontent.com/5800726/34863393-2e56da34-f795-11e7-8cec-8cfad0183c0d.png)

---

![image](https://user-images.githubusercontent.com/5800726/34863407-3f6c2e50-f795-11e7-8913-2efc0cd302f2.png)

---

![image](https://user-images.githubusercontent.com/5800726/34863432-5caf1306-f795-11e7-94b4-86d698396f13.png)

---

![image](https://user-images.githubusercontent.com/5800726/34863443-6bbfffd6-f795-11e7-9ca9-1a1a62da7bb6.png)


---

*Any of these designs can be converted into the t-shirts, cups, and in building plates too :laughing:*

**Comment by 94151** [comment]
few mockups for Jugaadfest logo here
![7c81a8e2-ac41-407e-b1db-15406635a3d0](https://user-images.githubusercontent.com/17044000/37520207-2cb6b2a6-2942-11e8-808c-cb97a32ec664.png)
![8b2983f6-b81d-4886-bd07-28599420b9e7](https://user-images.githubusercontent.com/17044000/37520208-2d1b38ca-2942-11e8-88c9-ed057f76166e.png)
![3330619d-a5fd-4b12-b0f3-562e0d6f8ba8](https://user-images.githubusercontent.com/17044000/37520209-2d7bfdc2-2942-11e8-9ed4-291907e70099.png)
![c6c4b532-20ea-4276-916c-0845c9357a54](https://user-images.githubusercontent.com/17044000/37520211-2ddfc0d2-2942-11e8-8c8d-36f1a8c95ea9.png)
![c414addb-c2df-4431-8738-d38f4bf8eb8a](https://user-images.githubusercontent.com/17044000/37520213-2e3cdab0-2942-11e8-8777-4478a70099ac.png)

**Comment by mariobehling** [comment]
Thanks a lot to everyone! A lot of awesome ideas were put together here and it took us a while to follow up. However, time has come. Jugaadfest 2018 will take place on Sep. 30 in Hyderabad. Check out https://jugaadfest.com

Great job all of you! @YashKumarVerma Your colorful idea of Jugaad is very convincing and we are taking this as a basis for the logo.

Please help to improve this here and provide a non-pathified SVG logo as well. Keep up the good work. Please follow up at https://github.com/fossasia/jugaadfest-artwork :smile: 

The first issue is here: https://github.com/fossasia/jugaadfest-artwork/issues/1
